FluentAvalonia/Styling/ControlThemes/BasicControls/CalendarStyles.axaml

```diff
@@ -392,7 +392,7 @@
                                     <ContentPresenter Name="Text"
                                                       Foreground="{TemplateBinding Foreground}"
                                                       Background="{TemplateBinding Background}"
-                                                      BorderBrush="{DynamicResource }"
+                                                      BorderBrush="{DynamicResource CalendarViewBorderBrush}"
```

Comment:
This should be `CalendarViewNavigationButtonBorderBrush` per the [21h1 styles](https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/CommonStyles/CalendarView_themeresources_21h1.xaml). Probably can put that up in a Setter and TemplateBinding it in.

Reply:
> Probably can put that up in a Setter and TemplateBinding it in.

I thought the same, but wasn't sure if you'd like it. Pushed the requested changes.